bundles/org.openhab.persistence.jdbc/src/main/feature/feature.xml

```diff
@@ -52,4 +52,10 @@
 		<bundle start-level="80">mvn:org.openhab.addons.bundles/org.openhab.persistence.jdbc/${project.version}</bundle>
 	</feature>
 
+	<feature name="openhab-persistence-jdbc-oracle" description="JDBC Persistence Oracle" version="${project.version}">
+		<configfile finalname="${openhab.conf}/services/jdbc.cfg" override="false">mvn:org.openhab.addons.features.karaf/org.openhab.addons.features.karaf.openhab-addons-external/${project.version}/cfg/jdbc</configfile>
+		<feature prerequisite="false" dependency="false">openhab-runtime-base</feature>
+		<bundle start-level="80">mvn:org.openhab.osgiify/com.oracle.database.jdbc.ojdbc11/23.5.0.2407</bundle>
```

Comment:
```suggestion
		<bundle start-level="80">mvn:org.openhab.osgiify/com.oracle.database.jdbc.ojdbc11/${oracle.version}</bundle>
```

Couldyou check if this is available, if so it would be better maintainable.

Reply:
Building just the addon with the `oracle.version` variable works and feature verification passes. I will change it for all features. If the online build fails, I can revert.